compiler/rustc_middle/src/middle/mod.rs

```diff
@@ -12,7 +12,7 @@ pub mod lib_features {
     #[derive(HashStable, TyEncodable, TyDecodable)]
     pub enum FeatureStability {
         AcceptedSince(Symbol),
-        Unstable,
+        Unstable(Option<Symbol>),
```

Comment:
it's nicer I think if this is a named variant with `old_name` in it. I wouldn't know what this meant otherwise. AcceptedSince is clearer in that regard so is fine staying the way it is. Ping me after that and I'll r+ @rusbot author